resources/release-notes.md

```diff
@@ -426,6 +449,7 @@ integration and the payer.
 [swish-other-features]: /payments/swish/other-features
 [swish-redirect]: /payments/swish/redirect
 [swish-seamless-view]: /payments/swish/seamless-view
+[seamless-view-events-card]: payments/card/other-features#seamless-view-events
```

Comment:
Codacy is saying this link doesn't work.

Reply:
corrected!